components/elm/cime_config/config_pes.xml

```diff
@@ -523,4 +523,33 @@
       </pes>
     </mach>
   </grid>
+  <grid name="l%ne4np4.*">
+    <mach name="chrysalis">
+      <pes compset="any" pesize="any">
+        <comment>elm on chrysalis: any compset on ne4 grid, 3x32x2 NODESxMPIxOMP</comment>
```

Comment:
Adding this to avoid over-decomposition run-time errors like
```
decompInit_lnd(): Number of clumps exceeds number of land grid cells 256 201
```
that can occur with otherwise default 4x32x2 PEs.